pkg/controllers/execution/execution_controller.go

```diff
@@ -185,6 +188,13 @@ func (c *Controller) syncToClusters(clusterName string, work *workv1alpha1.Work)
 	for _, manifest := range work.Spec.Workload.Manifests {
 		workload := &unstructured.Unstructured{}
 		err := workload.UnmarshalJSON(manifest.Raw)
+		reference, _ := ref.GetReference(c.Client.Scheme(), workload)
+		annotations := workload.GetAnnotations()
+		if annotations != nil && reference != nil {
+			if uid, uidExist := annotations[workv1alpha2.ObjectReferenceUIDLabel]; uidExist {
+				reference.UID = types.UID(uid)
```

Comment:
Currently the event sent by the `execution_controller` already contains `Kind`, `Name`, `Namespace`, just add the `UID` and the event will be bound to the workload. We can refer to the `searchEvents` method of `k8s.io/kubectl/pkg/describe/describe.go`. I think `resourceVerstion` is not necessary for events. I can not find 
 any relevant usage in k8s
@RainbowMango @Poor12

Reply:
It is possible that a third party event consumer may use `ResourceVersion`，we can add it later

Reply:
Thanks for confirming. Have you tested whether sync events are bound to workloads?

Reply:
Here is the result
![image](https://user-images.githubusercontent.com/14215449/172311745-30e54fa7-489c-411e-938d-3c3600557a6a.png)